docker/docker-compose.dev.yml

```diff
@@ -0,0 +1,47 @@
+# Docker Compose file for development
+version: "2"
+services:
+
+  db:
+    image: postgres:9.5.4
+    volumes:
+      - ../data/pgdata:/var/lib/postgresql/data/pgdata
+    environment:
+      PGDATA: /var/lib/postgresql/data/pgdata
+      POSTGRES_USER: acousticbrainz
+      POSTGRES_PASSWORD: acousticbrainz
+      POSTGRES_DB: acousticbrainz
+    ports:
+      - "15432:5432"
+
+  webserver:
+    build:
+      context: ..
+      dockerfile: ./docker/Dockerfile.dev
+    volumes:
+      - ../:/code
+      - ../data/app:/data
+    ports:
+      - "8080:8080"
+    depends_on:
+      - db
+      - acousticbrainz_cache
+
+  acousticbrainz_cache:
```

Comment:
We use redis for the container name in LB and MsB, renaming it for consistency might be a good idea.